tools/docsnapper/main.rs

```diff
@@ -338,9 +338,11 @@ fn process_tag(
 
     let size = width.and_then(|w| height.map(|h| (w, h)));
 
+    let scale_factor = attr.get("scale").and_then(|s| s.parse::<f32>().ok()).unwrap_or(1.0);
```

Comment:
Instead of adding this to all the tags, why not just default to a scale factor of two? I mean, is this really something we want configurable? :)

So I'd say:
 - Don't have a `scale` attribute, just use `2`.
 - If you want to have the attribute, make it at least default to `2` :)